packages/shared/src/components/squads/SquadHeaderMenu.tsx

```diff
@@ -70,6 +77,14 @@ export default function SquadHeaderMenu({
       },
     ];
 
+    if (!squad.currentMember && squad.public) {
+      list.unshift({
+        icon: <ContextMenuIcon Icon={LinkIcon} />,
+        action: () => trackAndCopyLink(),
+        label: 'Invitation link',
+      });
+    }
```

Comment:
I would rather have this be the first condition then using `unshift`, it will be vulnerable to order changes or accidental additions.

So I would propose:
- make a `list` empty array
- check the above condition for Invitation link and push if true
- push "Learn how squads work" after it without any condition.

Reply:
Like that the flow is always the same and new items added at the end.

Reply:
Good point, will do that :)